resources/knative-provisioner-natss/charts/knative-provisioner-natss/templates/natss.yaml

```diff
@@ -428,3 +342,91 @@ spec:
           name: config-logging
 
 ---
+{{- if not (.Capabilities.APIVersions.Has "natsschannels.messaging.knative.dev/v1alpha1") }}
+{{- if .Release.IsUpgrade }}
```

Comment:
Can you explain why this is needed?

Reply:
We need it to prevent upgrade from crushing when `CRD` is already defined.